app/views/report/bulk.html.erb

```diff
@@ -66,50 +66,77 @@ ul li{
-    <li><span class="bulk_button" id="republish_show" name="republish_show" onclick="$('#republish').show(400)">Republish</span>
-    <li><span class="bulk_button" id="show_tags" name="show_tags" onclick="$('#tag').show(400);get_tags();">Tags</span>
-  </ul>
-  <ul class="button_list">
-    <label>Metadata</label>
-    <li><span class="bulk_button" onclick="$('#create_desc_md').show(400)">Create MODS from label  </span>
-    <li><span class="bulk_button" onclick="$('#discoverable').show(400)">Check discoverability</span>
-    <li><span class="bulk_button" onclick="$('#schema_validate').show(400)">Validate MODS</span>
-    <li><span class="bulk_button" onclick="$('#refresh_metadata').show(400)">Refresh MODS</span>
-    <li><span class="bulk_button" onclick="$('#detect_duplicate_encoding').show(400)">Detect duplicate xml encoding</span>
-  </ul>
-  <ul class="button_list">
-    <label>Dangerous Stuff</label>
-    <li><span class="bulk_button" onclick="$('#purge').show(400)">Purge</span>
-    <li><span class="bulk_button" onclick="$('#release_hold').show(400)">Release for ingest</span>
-    <li><span class="bulk_button" onclick="$('#apply_apo_defaults').show(400)">Apply APO Defaults</span>
-    <li><span class="bulk_button" onclick="$('#add_workflow').show(400)">Add a Workflow</span>
-  </ul>
-
-  <%if current_user.is_admin%>
-    <ul class="button_list">
-      <label>Admin Only Actions</label>
-      <li><span class="bulk_button" onclick="$('#fix_provenance').show(400)">Add missing Provenance</span>
-      <!-- <li><span class="bulk_button" onclick="$('#remediate_mods').show(400)">Remediate MODS</span> disabled for now -->
-    </ul>
-  <%end%>
-
-    <ul class="button_list">
-      <label>Object Updates, you must prepare first!</label>
-      <li><span class="bulk_button" class="update_buttons" disabled id="show_source_id" name="show_source_id" onclick="$('#source_id').show(400);get_source_ids();">Source Id</span>
-      <li><span class="bulk_button" class="update_buttons" disabled onclick="$('#content_type').show(400)">Content type</span>
-      <li><span class="bulk_button" class="update_buttons" disabled onclick="$('#rights').show(400)">Object rights</span>
-      <li><span class="bulk_button" class="update_buttons" onclick="$('#set_collection').show(400)">Set collection</span>
-      <li><span class="bulk_button" class="update_buttons" onclick="$('#add_collection').show(400)">Add collection</span>
-    </ul>
+  <div id="buttons" class="row bulk-buttons">
+      <div id="versioning_heading" class="row bulk-buttons-row">
+	  <strong>Options that require versioning</strong>
+      </div>
+      <div id="versioning_buttons" class="row bulk-buttons-row">
+	  <div id="versioning_buttons_col_1" class="col-md-2">
+	      <div id="col_1_row_1" class="row">
+		  <span class="bulk_button btn-block" id="prepare" name="prepare" onclick="$('#open').show(400);">Prepare objects</span>
```

Comment:
Could these elements also be buttons? Then we could use the `btn` classes that bootstrap gives us.